roles/kafka/templates/cluster.yaml.j2

```diff
@@ -1,7 +1,7 @@
 apiVersion: kafka.strimzi.io/v1beta2
 kind: Kafka
 metadata:
-  name: my-cluster
```

Comment:
Does it make sense for the name of a cluster to be a variable?

Reply:
done